### DISCUSSION TITLE
How to translate &Remove

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#2839] [Translations]
Stellarium has phrases like &Remove. That means you can remove with alt + R. But if finnish language Poista has not r letter. How to translate this? There are many other examples too. Can I translate &Poista ?

I have translated Stellarium now 85 % to finnish language. Hope that it is completed before next major version. I have asked terms on astronomy forum. So I hope that they are quite correct translations.

**Comment by 10110111** [comment]
These accelerators are meant to differ in different languages. It's best to use the same accelerators for similar actions as used in other apps in your language, but if there's nothing well-known you can just choose the most convenient one. Just make sure it doesn't collide with other menu items, otherwise it'll no longer be a single key combo: you'll cycle between the menu items with the same accelerator and only activate the item you need by pressing <kbd>Return</kbd>.

**Comment by ghost** [comment]
Like in many other apps (I found many of them), in accord with what 10110111 writes, you can use a workaround for this issue. For your specific request try this: Poista (&R)
I know it doesn't look very nice but it sure should avoid any glitches from duplicate shortcuts.

**Reply by ilkant** [reply]
Thanks for the good suggestion! That's just fine.

**Reply by ilkant** [reply]
I completed finnish translation. Now it is ready to next version of Stellarium.  If I could get the .po file then I could test it as .qm file with Stellarium in my computer.

How to activate reviewer to review 9,683 unreviewed strings?

**Reply by alex-w** [reply]
@ilkant Please check Transifex

**Comment by ilkant** [comment]
I have just now completed all Stellarium translatable texts (from 8 % to 100 %). If you have not yet added translations to version 1.2 of Stellarium, you can do so. If you can't add anymore, I'll wait for the next version. One change has also been made to the WWW page, which had not yet appeared a short time ago.
